cnf-tests/Dockerfile

```diff
@@ -59,7 +73,7 @@ RUN yum install -y numactl-devel make gcc && \
     cp hwlatdetect /hwlatdetect && \
     cp cyclictest /cyclictest
 
-FROM quay.io/openshift/origin-oc-rpms:4.16 AS oc
+FROM quay.io/openshift/origin-oc-rpms:4.19 AS oc
 
 # Final image
 FROM centos:7
```

Comment:
I think you need to update this one also

Reply:
It was on purpose actually,  if we're changing to rhel9 then both `Dockerfile` and `Dockerfile.openshift` are identical. So one can be deleted